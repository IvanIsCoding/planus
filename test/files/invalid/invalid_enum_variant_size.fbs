enum Foo: uint8 {
  X = -1
}

enum Bar: uint8 {
  X = 256
}

enum Baz: int8 {
  X = 128
}