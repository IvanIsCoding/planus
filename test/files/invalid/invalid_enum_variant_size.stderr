[0m[1m[38;5;9merror[0m[1m: Integer is out of range for type u8 (range is [0; 255])[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/invalid_enum_variant_size.fbs:2:7
  [0m[34mâ”‚[0m
[0m[34m2[0m [0m[34mâ”‚[0m   X = [0m[31m-1[0m
  [0m[34mâ”‚[0m       [0m[31m^^[0m

[0m[1m[38;5;9merror[0m[1m: Integer is out of range for type u8 (range is [0; 255])[0m
  [0m[34mâ”Œâ”€[0m test/files/invalid/invalid_enum_variant_size.fbs:6:7
  [0m[34mâ”‚[0m
[0m[34m6[0m [0m[34mâ”‚[0m   X = [0m[31m256[0m
  [0m[34mâ”‚[0m       [0m[31m^^^[0m

[0m[1m[38;5;9merror[0m[1m: Integer is out of range for type i8 (range is [-128; 127])[0m
   [0m[34mâ”Œâ”€[0m test/files/invalid/invalid_enum_variant_size.fbs:10:7
   [0m[34mâ”‚[0m
[0m[34m10[0m [0m[34mâ”‚[0m   X = [0m[31m128[0m
   [0m[34mâ”‚[0m       [0m[31m^^^[0m

Error: Bailing because of errors
